charts/fybrik/templates/manager-deployment.yaml

```diff
@@ -135,9 +135,11 @@ spec:
               name: cert
               readOnly: true  
             {{- end }}     
-            - mountPath: /tmp/taxonomy
+            - name: data
+              mountPath: /data
+            - mountPath: {{ include "fybrik.printDataSubdir" ( tuple "taxonomy" ) }}
               name: fybrik-taxonomy
-            - mountPath: /tmp/adminconfig
+            - mountPath: {{ include "fybrik.printDataSubdir" ( tuple "adminconfig" ) }}
               name: fybrik-adminconfig
             {{- if .Values.manager.solver.image }}
             - mountPath: /tmp/solver
```

Comment:
also needs to be changed

Reply:
Thanks, I changed the mount dir of the solver also.